tests/integration/questionnaire/test_questionnaire_placeholders.py

```diff
@@ -30,3 +30,31 @@ def test_title_placeholders_rendered_in_summary(self):
         self.assertInUrl("/summary/")
         self.assertInBody("What is Kevin Bacon’s date of birth?")
         self.assertInBody("68 Abingdon Road, Goathill")
+
+    def test_placeholders_rendered_in_pages(self):
```

Comment:
A lot going on in this test, its been added to test the ordinals, but I wouldn't have known it out of context